packages/onboarding-ui/src/common/PlanFeatureIcon.tsx

```diff
@@ -7,7 +7,7 @@ const DescriptionIcon = ({
 }): ReactElement => (
   <svg
     width='13'
-    height='9'
+    height='s1'
```

Comment:
I don't get it

Reply:
Is it to put a new pattern on the size of the SVG, we shouldn't  use this directly on svg?